### DISCUSSION TITLE
Lightweight Tasmota Dashboard in PHP

### DISCUSSION THREAD

**Discussion by k0jdd** [arendst/Tasmota#18075] [Show and tell]
I know there are plenty of Tasmota Device managers out there.  I wanted something very lightweight to give me a quick overview of my Tasmota device status.

You will need to put these files in a directory on a local web server and then call the php script from any browser
Remember to Edit the.ini file to add your device IP addresses.

Screen Shot
![Dashboard](https://user-images.githubusercontent.com/46826320/221924013-95f64c7c-35e9-462e-b4b3-54a421e79ecd.jpeg)

Notes
The color of the word ON for Alarm state (an RGBW bulb) indicates the current bulb color

Timed out... indicates the device is most likely offline.  You can change the timeout in the ini file.

Zip file can downloaded from here --> 
https://k0jdd.netlogger.org/st/TasmotaDashboard.zip

Cheers

**Comment by barbudor** [comment]
I haven't look at your code yet
Just wondering if you are using MQTT or web polling

If you are using web polling please be gentle with tasmota which is optimized for mqtt. Web polling tends to use a lot of resources and tools such as TasmoAdmin make tasmota less reponsive

I have made such a web page in Node-red using only passive mqtt listening and parsing of LWT and Info1/Info2/Info3 messages

**Comment by k0jdd** [comment]
Hello and thanks for the reply.

As written, the script depends on a manual refresh in the browser to update the data.

For as often as I will look at this, that’s just fine.

It is good to know, however, that there could be detrimental effects from doing a lot of polling.

John
